tests/unit/s2n_client_key_share_extension_pq_test.c

```diff
@@ -656,7 +656,7 @@ int main()
 
                     /* Do NOT mark the highest priority available KEM group as mutually supported */
                     EXPECT_OK(s2n_set_all_mutually_supported_groups(server_conn));
-                    for (int i = 0; i < sizeof(server_conn->kex_params.mutually_supported_kem_groups); i++) {
+                    for (int i = 0; i < s2n_array_len(server_conn->kex_params.mutually_supported_kem_groups); i++) {
```

Comment:
Luckily this and line 720 are the only existing mistakes I found. From my testing, they currently pass without triggering an invalid memory access because they never iterate all the way through the loop. Currently, both always bail on the first kem group.